bofire/data_models/surrogates/botorch_surrogates.py

```diff
@@ -83,7 +83,8 @@ def _check_compability(self, inputs: Inputs, outputs: Outputs):
         if len(used_feature_keys) != len(inputs):
             raise ValueError("Unused features are present.")
 
-    @validator("surrogates")
+    @field_validator("surrogates")
+    @classmethod
     def validate_surrogates(cls, v, values):
         # validate that all surrogates are single output surrogates
         # TODO: this restriction has to be removed at some point
```

Comment:
Make an issue out of the TODO?

Reply:
At some point, currently I am too lazy :D